teps/0098-workflows.md

```diff
@@ -125,62 +54,102 @@ simplify the Tekton experience. It can provide a single place to declare all the
 
-1. Platform teams can control or restrict access to portions of the CI/CD workflow e.g. service accounts, access to secrets etc.
-1. A company has a centralized devops team rebuilding their CD system on Tekton components. They would like a way to abstract a lot of the boilerplate and setup involved when onboarding new teams to the system. Rather than write a new DSL they elect to use Tekton Workflows since it allows them to move more quickly than building and supporting their own solution would. 
+2. Builds triggered by branch changes and cron jobs
 
-## Requirements
+- The end user would like to build and release their project to their dev cluster whenever changes are detected on the main branch.
+- They don't want to set up an ingress on their cluster for security reasons, so they'd like to repeatedly poll the main branch for changes.
+- They would like build failures to be posted to Slack, but don't want their app developers to have to include this notification logic in their build Pipeline,
+especially since several other teams' build Pipelines should post to Slack too.
+- They would like to kick off a daily release of their project to the prod cluster based on the changes pushed to the dev cluster a week prior.
 
-<!--
-Describe constraints on the solution that must be met. Examples might include
-performance characteristics that must be met, specific edge cases that must
-be handled, or user scenarios that will be affected and must be accomodated.
--->
+3. CD system from scratch
 
-## Proposal
+A company has a centralized devops team rebuilding their CD system on Tekton components.
+They would like a way to abstract a lot of the boilerplate and setup involved when onboarding new teams to the system.
+Rather than write a new DSL they elect to use Tekton Workflows since it allows them to move more quickly than building and supporting their own solution would.
 
-<!--
-This is where we get down to the specifics of what the proposal actually is.
-This should have enough detail that reviewers can understand exactly what
-you're proposing, but should not include things like API designs or
-implementation.  The "Design Details" section below is for the real
-nitty-gritty.
--->
+4. Platform builder
+
+A company would like to make it easier for end users to run Tekton on its platform.
+They would like to be able to extend Tekton Workflows to allow end users to trigger PipelineRuns from repositories connected
+via their platform, view logs via their log storage, load secrets from their secret storage system, etc, even though
+first-class support for their products is not built directly into Tekton Workflows.
+
+## Requirements
+
+- End users can set up an entire end to end CI pipeline (from source back to run results) using only Tekton 
+- End users can create a repo connection once and use it for multiple Workflows. For example, they might want both a CI and a CD Workflow for a given repo.
```

Comment:
A bit confused about this one - this feels a bit more like an implementation detail ?

Reply:
added some clarifying language-- I think it would be nice if they only had to provide one set of repo configuration (e.g. access tokens) and reuse that configuration. I agree the actual connection is an implementation detail.